docs/auth0_apps_create.md

```diff
@@ -21,29 +21,29 @@ auth0 apps create [flags]
   auth0 apps create --name myapp 
   auth0 apps create --name myapp --description <description>
   auth0 apps create --name myapp --description <description> --type [native|spa|regular|m2m]
-  auth0 apps create --name myapp --description <description> --type [native|spa|regular|m2m] --reveal
+  auth0 apps create --name myapp --description <description> --type [native|spa|regular|m2m] --reveal-client-secret
```

Comment:
Note that when passing the `--json` flag, the reveal flag will not only show the client secret, but the signing keys as well: https://github.com/auth0/auth0-cli/blob/v1/internal/display/apps.go#L244

Would it make sense to rename it to something a little more generic, like `--reveal-secrets`?

Reply:
Definitely makes sense 👍🏻 , updated in [`44d27bf` (#591)](https://github.com/auth0/auth0-cli/pull/591/commits/44d27bf2f7123d8e6f54ddba6134bc32c8c37fdc).